### PR TITLE
C3D-tensorflow Third-party Implementation

### PR DESCRIPTION
**Pull request by cc786537662**
I add a cite with my C3D-tensorflow Implementation in the README ,anyone wants to do more job on C3D with tensorflow would find it helpful.
I also provied tensorflow C3D  pre-trained models converted from your official caffemodel.

and there is about 5% accuracy margin between this implement and your official C3D, i think more train augment and more careful finetuning can make up this margin.

### CODE REVIEW

**Comment by dutran** [comment]
Thanks  @cc786537662
